common/app/com/commercetools/sunrise/ctp/project/ProjectModule.java

```diff
@@ -0,0 +1,16 @@
+package com.commercetools.sunrise.ctp.project;
+
+import com.google.inject.AbstractModule;
+import io.sphere.sdk.projects.Project;
+
+import javax.inject.Singleton;
+
+public final class ProjectModule extends AbstractModule {
```

Comment:
The name clearly states that it provides a project, but as an user you have to look at the source code to see what this module provides. What do you think about adding some javadoc here?